pkg/kubernetes/provider_test.go

```diff
@@ -160,7 +151,7 @@ func Test_GenerateProviderNodeAffinity(t *testing.T) {
 			wantNSR:           []corev1.NodeSelectorRequirement{},
 		},
 		{
-			name: "one existing provider, default provider",
+			name: "one existing provider, default/ubuntu provider",
```

Comment:
```suggestion
			name: "one existing provider, default provider",
```
I think we can keep this as the `default` provider in case we ever add an `ubuntu` provider